portfolio/src/main/webapp/script.js

```diff
@@ -27,13 +27,20 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-async function getHello() {
+async function loadComments() {
   const response = await fetch('/data');
-  const message = await response.text();
-  const helloButton = document.getElementById('hello');
-  helloButton.innerText = message;
-}
+  const oldCommentArray = await response.text();
+  //Remove quotes and brackets from comments
```

Comment:
since this is JSON format, I suppose there should be a lib in JS to handle JSON?